### DISCUSSION TITLE
Not able to install the web application!

### DISCUSSION THREAD

**Discussion by awescure** [globaleaks/globaleaks-whistleblowing-software#4147] [Q&A] [answered]
I have asked my hoster's support to install the Global Leaks web application, as I am “only” a user and max. 1st level. I received the following reply: 
"Unfortunately, we had to realize that the desired installation of the Global Leaks software is not possible natively in Plesk, as both Plesk and Global Leaks claim port 8443 for themselves. As an installation is therefore only possible 'bypassing Plesk', we are unfortunately unable to install the software functionally"
Are there any suggestions that fit the goal of installing the web application quickly (please note I am a user and max. 1 level)?

**Comment by evilaliv3** [comment, marked as answer]
Hello @awescure 

Actually to run globaleaks you need a dedicated virtual or physical server running ubuntu or debian on which to run only globaleaks.
